resources/views/projects/create.blade.php

```diff
@@ -0,0 +1,252 @@
+							@endforeach
+						@endif
+					</select>
+				</div>
+				<div class="two wide field">
+					<br>
+					<p id="or">or</p>
+				</div>
+				<div class="seven wide field">
+					<input type="text" name="createTame" placeholder="New Team Name">
+				</div>
+			</div>
+		</div> --}}
+		<!-- Associated Course -->
+		<div class="field">
+			<label for="courses">Associated Course</label>
+			<select name="courses" class="ui search dropdown">
+				<option value="">Web Development Class</option>
+				@if(isset($courses))
+					@foreach($courses -> all() as $course)
+						<option value={{$course['id']}}>
+						{{$course['name']}}</option>
+					@endforeach
+				@endif
+			</select>
+		</div>
+		<!-- Category -->
+		<div class="field">
+			<label for="category">Category</label>
+			<select name="category" class="ui search dropdown">
+				<option value="">Finance</option>
+				@if(isset($categories))
+					@foreach($categories -> all() as $category)
+						<option value={{$category['id']}}>{{$category['name']}}</option>
+					@endforeach
+				@endif
+			</select>
+		</div>
+		<!-- Skillset -->
+		<div class="field">
+			<label for="skillsets">Skillsets</label>
+			<select name="skillsets" multiple="" class="ui fluid dropdown">
+				<option value="">Java, HTML</option>
+				@if(isset($skillsets))
+					@foreach($skillsets -> all() as $skillset)
+						<option value={{$skillset['id']}}>
+						{{$skillset['name']}}</option>
+					@endforeach
+				@endif
+			</select>
+		</div>
+		<!-- Milestone -->
+		<div class="field">
+			<label for="milestone">Public Milestone</label>
+			<input type="text" name="milestone" placeholder="Design">
+		</div>
+		<!-- Short Description -->
+		<div class="field">
+			<label for="shortDescription">Brief Description</label>
+			<textarea name="shortDescription" rows="2" placeholder="The project is a web page for personal financial organization"></textarea>
+		</div>
+		<!-- Long Description -->
+		<div class="field">
+			<label for="longDescription">Detailed Description</label>
+			<textarea name="longDescription" placeholder="The project consists of a single page application where users can sign up or login. It includes..."></textarea>
+		</div>
+		<!-- Video Pitch -->
+		<div class="field">
+			<label for="videoPitch">Pitch Video</label>
+			<input type="text" name="videoPitch" placeholder="https://youtube.com/watch?v=238028302">
+		</div>
+		<!-- Register Button -->
+		<button id="registerButton" type="submit" class="ui primary submit button">Register Project</button>
+		<!-- Error Message -->
+		<div class="ui error message"></div>
+	</form>
+</div>
+@section('scripts')
+<script>
+	/* Upload new image */
+	$(".imgUploader").click(function(event) {
+		event.preventDefault();
+		$("#imgInput").click();
+	});
+	function validateImage(file) {
+		const maxImageSize=1000000; // 1MB
+		if((file.type == "image/png" || file.type == "image/jpg") && file.size <= maxImageSize) {
```

Comment:
Esto se puede simplificar a `return (file.type == "image/png) || .. <rest of condition>` y eliminar el `if... else`

Reply:
> Hay un ejemplo de todo lo de image validation que ya está aprobada en [/resources/views/user/edit.blade.php](https://github.com/ProyectoIntegrador2018/BlueMarket/blob/dev/resources/views/user/edit.blade.php). Maybe la puedes usar de referencia? @marcegarzab 

Same here.